src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1533,7 +1539,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Private ReadOnly Property IMethodBindingExpression_IsVirtual As Boolean Implements IMethodBindingExpression.IsVirtual
             Get
-                Return Me.Method IsNot Nothing AndAlso (Me.Method.IsOverridable OrElse Me.Method.IsOverrides OrElse Me.Method.IsMustOverride) AndAlso Not Me.SuppressVirtualCalls
+                Dim method As IMethodSymbol = Me.Method
+                Return method IsNot Nothing AndAlso Me.ReceiverOpt IsNot Nothing AndAlso (method.IsVirtual OrElse method.IsAbstract OrElse method.IsOverride) AndAlso Not Me.ReceiverOpt.SuppressVirtualCalls
             End Get
```

Comment:
The code is the same as in `IInvocationExpression_IsVirtual` above, consider create a helper.